clarify case sensitive attribute values (#27658)
mdn/content: files/en-us/web/css/attribute_selectors/index.md

The CSS **attribute selector** matches elements based on the element having a given attribute explicitly set, with options for defining an attribute value or substring value match.

> The case sensitivity of attribute names and values depends on the document language. In HTML, attribute names are case insensitive, as are spec-defined {{glossary("enumerated")}} values. The [list of case-insensitive CSS attribute values](https://html.spec.whatwg.org/multipage/semantics-other.html#case-sensitivity-of-selectors) is enumerated in the HTML spec. For these attributes, the attribute value in the selector is case-insensitive, even if it's an invalid value or if the attribute is not a valid value for the element on which it is set.

Reviewer: - Can we say "listed" instead of "enumerated" for simplicity? :)
- This should be HTML attributes, right?

```suggestion
The case sensitivity of attribute names and values depends on the document language. In HTML, attribute names are case insensitive, as are spec-defined {{glossary("enumerated")}} values. The [case-insensitive HTML attribute values](https://html.spec.whatwg.org/multipage/semantics-other.html#case-sensitivity-of-selectors) are listed in the HTML spec. For these attributes, the attribute value in the selector is case-insensitive, regardless of whether the value is invalid or the attribute for the element on which it is set is invalid.
```
